provider/backingchain/blockcommand_base.py

```diff
@@ -258,10 +258,14 @@ def prepare_secret_disk(self, image_path, secret_disk_dict=None):
         vmxml.sync()
 
     @staticmethod
-    def clean_file(file_path):
+    def clean_file(file_path, session=None):
         """
         Clean file
+
         :params file_path: the path of file to delete
+        :params session: vm session
         """
+        if session:
+            session.cmd("rm -f %s" % file_path)
         if os.path.exists(file_path):
```

Comment:
should you update the 2nd 'if' to 'elif'? 
```suggestion
        if session:
            session.cmd("rm -f %s" % file_path)
        elif os.path.exists(file_path):
```

Reply:
还真是

Reply:
Thanks your patience ,yingshun Done :D,